client/routes/signin.vue

```diff
@@ -1,61 +1,113 @@
 <template>
-  <section>
-    <div class="signin">
-      <div v-if="currentUser">
-        <img :src="currentUser.picture" width="200" alt="user pic" />
-        <dl>
-          <dt>Name</dt>
-          <dd>{{ currentUser.name }}</dd>
-          <dt>Email</dt>
-          <dd>{{ currentUser.email }}</dd>
+  <section class="signin">
+    <div class="signin-form">
+      <img
+        :src="
+          currentUser
+            ? currentUser.picture
+            : 'https://seeklogo.com/images/S/snapchat-logo-47531E7AE8-seeklogo.com.png'
```

Comment:
can we use the temporal logo here as the placeholder?

Reply:
oops